mysql-test/t/percona_slow_general_log_names.test

```diff
@@ -0,0 +1,3 @@
+--replace_regex /\.\d+//
```

Comment:
Move MTR test case to `percona` test suite and rename it to `slow_general_log_names`

Reply:
Done